### PR TITLE
Add FIP-73 proposal to spin up a LUSD->DAI swapper

### PR DESCRIPTION
**Pull request by eswak**
Forum discussion: https://tribe.fei.money/t/fei-peg-policy-changes/3906
Snapshot: https://snapshot.org/#/fei.eth/proposal/0x825420b492363969de1d728655904181d92df2984631a232533e6a86b2125ee3

Immediately swap 20M LUSD for 20M DAI using Curve to re-fill the DAI reserves and allow DAI redemptions.

Create a new swapper for LUSD->DAI whitelisted as a safe address to allow the PCVGuardian to exchange LUSD for DAI using an LBP swapper like used for the buybacks.

The DAI received by swapping will be sent to Compound, where the dripper can pull it to the DAI PSM.

### CODE REVIEW

File: proposals/description/fip_73_swap.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        '{compoundDaiPCVDeposit}'
+      ],
+      description: 'Swap 20M LUSD on Curve for at least 19.96M DAI and send to Compound DAI deposit'
+    },
+    {
+      target: 'compoundDaiPCVDeposit',
+      values: '0',
+      method: 'deposit()',
+      arguments: [],
+      description: 'Deposit 20M DAI on Compound'
```
Comment (ElliotFriedman) [inline proposals/description/fip_73_swap.ts:38]: why are we depositing to compound? can't we just send straight to dai psm?

Comment (eswak) [reply proposals/description/fip_73_swap.ts:38]: it may never go through the dripper / psm if there are no redemptions

Comment (eswak) [reply proposals/description/fip_73_swap.ts:38]: anyway a permissionless call on the PSM will re-allocate the extra DAI to compound

File: proposals/dao/fip_73_swap.ts [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+  await lusdToDaiLensDai.deployTransaction.wait();
+
+  logging && console.log('BPTLens for DAI in swapper pool: ', lusdToDaiLensDai.address);
+
+  const lusdToDaiLensLusd = await BPTLensFactory.deploy(
+    addresses.lusd, // token reported in
+    noFeeLusdDaiLBPAddress, // pool address
+    addresses.oneConstantOracle, // reportedOracle
+    addresses.oneConstantOracle, // otherOracle
```
Comment (Joeysantoro) [inline proposals/dao/fip_73_swap.ts:94]: LUSD and DAI have oracles, why use constant?

File: proposals/dao/fip_73_swap.ts [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+  // 3.
+  const tx2 = await lusdToDaiLBPSwapper.init(noFeeLusdDaiLBPAddress);
+  await tx2.wait();
+
+  // 4.
+  const BPTLensFactory = await ethers.getContractFactory('BPTLens');
+  const lusdToDaiLensDai = await BPTLensFactory.deploy(
+    addresses.dai, // token reported in
+    noFeeLusdDaiLBPAddress, // pool address
+    addresses.oneConstantOracle, // reportedOracle
```
Comment (Joeysantoro) [inline proposals/dao/fip_73_swap.ts:80]: LUSD and DAI have oracles, why use constant?

**Pull request closed without merging**